fanuc_driver/launch/motion_streaming_interface.launch

```diff
@@ -3,17 +3,17 @@
 -->
 <launch>
   <!-- IP of robot (or PC running simulation) -->
-  <arg name="robot_ip" />
+  <arg name="robot_ip" doc="IP of controller" />
 
   <!-- J23_factor: set to correct factor to compensate for J23 coupling:
               -1 : negative coupling (J3' = J3 - J2)
                0 : no coupling       (J3' = J3)
                1 : positive coupling (J3' = J3 + J2)
   -->
-  <arg name="J23_factor" />
+  <arg name="J23_factor" doc="Compensation factor for joint 2-3 coupling (-1, 0 or 1)" />
 
   <!-- Load the byte-swapping version of robot_state if required -->
-  <arg name="use_bswap" />
+  <arg name="use_bswap" doc="Iff true, driver will perform byte-swapping on incoming data stream" />
```

Comment:
This is the mathematical / logical [if and only if](https://en.wikipedia.org/wiki/If_and_only_if). So not a typo, more a habit ("only if this is set to true, will ..").